test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -28,7 +28,6 @@ static int full_test;
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
 
 struct suite_context_s {
-	odp_bool_t packet;
 	odp_crypto_op_mode_t op_mode;
 	odp_crypto_op_mode_t pref_mode;
```

Comment:
`pref_mode` is unused as well?

Reply:
Yes, it is now unused. Will remove it in v2.